src/Common/FitOnFhir.Common/Services/UsersServiceBase.cs

```diff
@@ -45,6 +46,10 @@ public async Task EnsurePatientAndUser(string platformName, string identifier, s
                 user.AddPlatformUserInfo(new PlatformUserInfo(platformName, identifier, DataImportState.ReadyToImport));
                 await _usersTableRepository.Insert(user, cancellationToken);
             }
+
+            QueueFitnessImport(user, cancellationToken);
         }
+
+        public abstract void QueueFitnessImport(User user, CancellationToken cancellationToken);
```

Comment:
The return type here should be a Task. This calls an async Task which needs to be awaited.